eclipsePlugin/test/de/tobject/findbugs/reporter/ThrottledProgressMonitorTest.java

```diff
@@ -0,0 +1,52 @@
+package de.tobject.findbugs.reporter;
```

Comment:
That's the problem (one from many) with maven. There was a life before it, but people tend to forget it :-) 

One of (from many) really stupid things maven people did was to break the habit to have clean modular code, now everything need to be in one place, this is awful.

Production code should be always separated from tests, and thanks to the fact Eclipse was developed before maven and without maven, they also had a very good choice to have clean separated test bundle projects and clean separated production bundle projects.